src/20_Components/amp-next-page.html

```diff
@@ -0,0 +1,147 @@
+<!---
+  {
+    "preview": "default",
+    "experiments": ["amp-next-page"]
+  }
+--->
+<!--
+  ## Introduction
+
+  [amp-next-page](https://github.com/ampproject/amphtml/blob/master/extensions/amp-next-page/amp-next-page.md)
```

Comment:
we should link to ampproject.org instead of the github project [https://www.ampproject.org/docs/reference/components/amp-next-page](https://www.ampproject.org/docs/reference/components/amp-next-page)